app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError("NotVaccinatedError")
+        elif visitor["vaccine"].get("expiration_date") < datetime.date.today():

```

Comment:
Now possible compare None and date
if a vaccine exist then expiration_date always exist
Just do: visitor["vaccine"]["expiration_date"]

Reply:
Fixed v.3